crossfit/backend/torch/op/base.py

```diff
@@ -67,19 +81,25 @@ def call(self, data, partition_info=None):
         all_outputs_ls = []
         for output in loader.map(self.model.get_model(self.get_worker())):
             if isinstance(output, dict):
-                if self.model_output_col not in output:
-                    raise ValueError(f"Column '{self.model_output_col}' not found in model output.")
-                output = output[self.model_output_col]
-
+                if self.model_output_cols:
+                    output = {col: output[col] for col in self.model_output_cols if col in output}
+                if len(output) == 1:
+                    output = list(output.values())[0]
+                elif len(output) > 1 and self.model_output_cols is None:
```

Comment:
there is one conditional missing i.e `len(output) == 0` i.e `output` is a dict with keys that don't exist in `model_output_cols`

Reply:
Let me create a loud failure for that case

Reply:
Done